trunk/SUAVE/Methods/Geometry/Two_Dimensional/Cross_Section/Airfoil/import_airfoil_polars.py

```diff
@@ -6,106 +6,91 @@
 #           Sep 2020, M. Clarke 
 #           May 2021, R. Erhard
 #           Nov 2021, R. Erhard
-#           Jul 2022, R. Erhard
 
 # ----------------------------------------------------------------------
 #  Imports
 # ----------------------------------------------------------------------
-from SUAVE.Core import Data  
+from SUAVE.Core import Data, Units 
 import numpy as np
 
 ## @ingroup Methods-Geometry-Two_Dimensional-Cross_Section-Airfoil
-def import_airfoil_polars(airfoil_polar_files, airfoil_names):
-    """This imports airfoil polars from a text file output from XFOIL or from a 
-    text file containing the (alpha, CL, CD) data from other sources.
+def  import_airfoil_polars(airfoil_polar_files):
+    """This imports airfoil polars from a text file output from XFOIL or Airfoiltools.com
     
     Assumptions:
     Input airfoil polars file is obtained from XFOIL or from Airfoiltools.com
-
     Source:
-    N/A
-
+    http://airfoiltools.com/
     Inputs:
     airfoil polar files   <list of strings>
-
     Outputs:
     data       numpy array with airfoil data
-
     Properties Used:
     N/A
     """      
+     
+    # number of airfoils   
+    num_polars                   = 0 
+    n_p = len(airfoil_polar_files)
+    if n_p < 3:
+        raise AttributeError('Provide three or more airfoil polars to compute surrogate') 
+    num_polars            = max(num_polars, n_p)       
     
-    # number of airfoils 
-    num_airfoils = len(airfoil_polar_files)  
-    
-    num_polars   = 0
-    for i in range(num_airfoils): 
-        n_p = len(airfoil_polar_files[i])
-        if n_p < 3:
-            raise AttributeError('Provide three or more airfoil polars to compute surrogate')
-        
-        num_polars = max(num_polars, n_p)
+    # create empty data structures 
+    airfoil_data = Data()
+    dim_aoa      = 89 # this is done to get an AoA discretization of 0.25
```

Comment:
don't hardcode this

Reply:
This was also reverting back to the old scripts...nevertheless, I will make it an input